### ISSUE TITLE
ktf failed to run admin panel finder

### ISSUE THREAD

**Issue by Keerahere** [PowerScript/KatanaFramework#24]
+[ktf](web/cp.finder):run
[run] The module was launched...
[inf] Sat Oct 6 17:01:57 2018
[inf] Loading file [files/db/commons-dir-admin.tbl]
[stp] [1] Step : Starting Brute Force...
[Socket] [war]([Errno -2] Name or service not known)

**Comment by trollmad3** [comment]
Did you include `https` or `http`?

**Comment by souli0** [comment]
Did you manage to fix it? i have the same issue

**Comment by Artook10** [comment]
Yes, you change the SSL to false or back to true if you have changed it in
the past. Some servers will accept one or the other SSL so if you have true
selected
and the server you are scraping is false the error is returned.

**Comment by souli0** [comment]
+[ktf](web/ap.finder):set ssl false
             ���--------> ssl = false
  +[ktf](web/ap.finder):run
   [run] The module was launched...
   [inf] Thu Feb 21 06:39:14 2019
   [inf] Loading file [files/db/commons-dir-admin.tbl]
   [stp] [1] Step : Starting Brute Force...
 [Socket]  [war]([Errno -2] Name or service not known)

  +[ktf](web/ap.finder):set ssl true
             ���--------> ssl = true
  +[ktf](web/ap.finder):run
   [run] The module was launched...
   [inf] Thu Feb 21 06:39:30 2019
   [inf] Loading file [files/db/commons-dir-admin.tbl]
   [stp] [1] Step : Starting Brute Force...
 [Socket]  [war]([Errno -2] Name or service not known)

sad it didn't work thanks to have tried.

**Comment by Artook10** [comment]
Welcome hoped that would have worked for you.

**Comment by jorgegso** [comment]
> +[ktf](web/cp.finder):run
> [run] The module was launched...
> [inf] Sat Oct 6 17:01:57 2018
> [inf] Loading file [files/db/commons-dir-admin.tbl]
> [stp] [1] Step : Starting Brute Force...
> [Socket] [war]([Errno -2] Name or service not known)


I had the same error with a target on a localhost and I solved it by separating the port from the localhost:
set target localhost
set port 3000
I don't know if it helps

**Comment by Baxterthedog1** [comment]
@RedToor

**Comment by parthk2510** [comment]
Is the issue solved ? Any solution provider ?

**Comment by Aholicknight** [comment]
> Is the issue solved ? Any solution provider ?

@parthk2510 the creator of the repo @RedToor has not been active for a long time so **the issue has not been resolved**.
